connectivity/check/deployment.go

```diff
@@ -32,9 +32,10 @@ const (
 
 	perfHostNetNamingSuffix = "-host-net"
 
-	clientDeploymentName  = "client"
-	client2DeploymentName = "client2"
-	clientCPDeployment    = "client-cp"
+	clientDeploymentName         = "client"
+	client2DeploymentName        = "client2"
+	clientCPDeployment           = "client-cp"
+	clientExternalNodeDeployment = "client-external-node"
```

Comment:
Do we actually need this external client? We already have a pod running on that external node. You can check the `outsideToNodePort` test case.

Reply:
ah thanks :heavy_check_mark: